docs/user-guide/station-module-configuration.md

```diff
@@ -1,11 +1,20 @@
-# Basics Station IoT Edge Module configuration parameters
+# 'LoRaBasicsStationModule' IoT Edge Module configuration parameters
```

Comment:
This will be the title used on the sidebar of the docs - is this intended?

Reply:
Yes, anything against it?

Reply:
No no, just wanted to check, as it is sometimes not clear where the titles comes from (they can be set in different locations)